packages/cosmic-swingset/README.md

```diff
@@ -148,25 +146,33 @@ $ make scenario0-setup
 $ make scenario0-run-client
 ```
 
-Alternatively, running the solo node from a Docker image and no local source code is described in the [top section](#agorics-cosmic-swingset).  
+Alternatively, running the solo node from a Docker image and no local source code is described in the [Testnet Tutorial](#testnet-tutorial).
 
 Now go to http://localhost:8000/ to interact with your new solo node.
 
 Learn more about ERTP [here](https://agoric.com/documentation/ertp/guide/). 
 
-To see the contracts you've uploaded [as per the README](lib/ag-solo/contracts/README-contract.md), try:
+To see the contracts you've uploaded, try:
+
+```js
+E(home.uploads).list()
+```
 
+And to interact with them:
 ```js
-home.uploads~.list()
-home.uploads~.get('encouragementBot')~.spawn()~.encourageMe('Person')
+E(
+  E(
+    E(home.uploads).get('encouragementBot')
+  ).spawn()
+).encourageMe('Person')
```

Comment:
This section should be deleted.

Reply:
Just this code block, or the one above it as well, or even more than that?

Reply:
The part that I highlighted: `To see the contracts ...` up to and including `).encourageMe('Person')`.